File: Plugin/ShadowWarden/src/main/java/dev/didelfo/shadowWarden/manager/connections/websocket/components/MessageWS.java
package dev.didelfo.shadowWarden.manager.connections.websocket.components;

public class MessageWS {

    private String data;
    private String signature;
    private String id;

    public MessageWS(String data, String signature, String id) {
        this.data = data;
        this.signature = signature;
        this.id = id;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getSignature() {
        return signature;
    }

    public void setSignature(String signature) {
        this.signature = signature;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: APP/ShadowWarden/app/src/main/java/dev/didelfo/shadowwarden/connection/websocket/components/MessageWS.kt
package dev.didelfo.shadowwarden.connection.websocket.components

import com.google.gson.annotations.SerializedName

data class MessageWS(
    var data: String,
    var signature: String,
    var id: String
) {
    override fun toString(): String = "{\"data\": \"${data}\",  \"signature\": \"${signature}\", \"id\": \"${id}\"}"
}